client/js/widgets/widget.js

```diff
@@ -357,6 +357,21 @@ export class Widget extends StateManaged {
     super.setPosition(x, y, z);
   }
 
+    setText(text, mode, debug, problems) {
+    if (this.p('text') !== undefined) {
+      if(mode == 'inc' || mode == 'dec')
+        this.p('text', (parseInt(this.p('text')) || 0) + (mode == 'dec' ? -1 : 1) * text);
+      else if(Array.isArray(text))
+        this.p('text', text.join(', '));
+      else if(typeof text == 'string' && text.match(/^[-+]?[0-9]+(\.[0-9]+)?$/))
+        this.p('text', +text);
+      else
+        this.p('text', text);
+      if (debug) console.log(`changing ${this.id} ${mode} ${text}`)
```

Comment:
Not sure where this debug message comes from but it seems unnecessary to me and weird that it gets toggled by the `debug` property.